kasa/iot/modules/led.py

```diff
@@ -30,3 +30,8 @@ def led(self) -> bool:
     async def set_led(self, state: bool):
         """Set the state of the led (night mode)."""
         return await self.call("set_led_off", {"off": int(not state)})
+
+    @property
+    def is_supported(self) -> bool:
+        """Return whether the module is supported by the device."""
+        return "led_off" in self.data
```

Comment:
Unrelated change.

Reply:
This is related because previously the led module was determined as not supported because the default `is_supported` implementation assumes it's data is in a module attribute outside of sys info.  We've only noticed this in this PR as `modules` now returns only the supported ones.